### ISSUE TITLE
Issue for new release of `gliderlabs-sigil` package

### ISSUE THREAD

**Issue by ronazst** [dokku/dokku#6899]
### Description of problem

The new released `gliderlabs-sigil` package having Permission deined issue due to no execute `x` permission for `/usr/bin/sigil`.

After grant `x` permission to the sigil file, then it will has new error:
```
/usr/bin/sigil: line 1: syntax error near unexpected token `newline'
/usr/bin/sigil: line 1: `!<arch>'
```

User will see the following error, if they just upgrade the package:
```
/var/lib/dokku/plugins/enabled/nginx-vhosts/install: line 116: /usr/bin/sigil: Permission denied
```

### Steps to reproduce

Debian bookworm:
```
apt update && apt upgrade
```

### dokku report $APP_NAME

this not related to any app, it's about dokku install.

**Comment by ronazst** [comment]
Any people having the same issue can try pin the `gliderlabs-sigil` to old version.

juse put the following content to file: `/etc/apt/preferences.d/pin-gliderlabs-sigil-dokku-fix` and reinstall `gliderlabs-sigil`

```
Package: gliderlabs-sigil
Pin: version 0.10.1
Pin-Priority: 1002
```

**Comment by josegonzalez** [comment]
I pulled the bad sigil release and will inspect what is going on that breaks releases.

I'll also think about how we can ensure bad dokku dependencies can't make it out to users.

**Comment by ronazst** [comment]
> I pulled the bad sigil release and will inspect what is going on that breaks releases.
> 
> I'll also think about how we can ensure bad dokku dependencies can't make it out to users.

seems you also own the gliderlabs-sigil package, how about add dokku usage as unit test in gliderlabs-sigil?

but for other dependencies not own by dokku team, how about just let it go? 
It's really hard to controll other dependencies, unless we also do "pin" dependency during dokku install, and update the pin version during upgrade.

BTW: the new gliderlabs-sigil missed the execute permission (chmod +x) which should also be another issue for package/install script.

**Comment by josegonzalez** [comment]
There is, but a release downstream doesn't (yet) trigger - rebuild (and it works be weird to hate downstream releases on dokku).

**Comment by kbuilds** [comment]
I actually ran into this the other day when I upgraded my dokku installation to `v0.34.4`. The upgrade was performed using `apt`. This prevented the core plugins from updating properly until I manually patched `/usr/bin/sigil` file.

As the OP said, this was causing an issue during the install file here:

https://github.com/dokku/dokku/blob/master/plugins/nginx-vhosts/install#L116

I am not sure where the upgrade process got the new `/usr/bin/sigil` file from, but the output from the `sigil` command was the exact same as the OP.
